model/src/main/java/org/mskcc/cmo/metadb/model/QcReport.java

```diff
@@ -3,6 +3,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
Considering removing `QcReport` completely pending decision on whether we want to change the type of `sampleMetadata.qcReports` to a `String` since that is how we store this member in the graph database anyway.